CRITICAL - 2021-04-12 07:09:28 --> Required parameter $from follows optional parameter $verbs
#0 G:\xampp\htdocs\ekinerja\system\Autoloader\Autoloader.php(365): CodeIgniter\Debug\Exceptions->errorHandler(8192, 'Required parame...', 'G:\\xampp\\htdocs...', 1017)
#1 G:\xampp\htdocs\ekinerja\system\Autoloader\Autoloader.php(365): require_once()
#2 G:\xampp\htdocs\ekinerja\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('G:\\xampp\\htdocs...')
#3 G:\xampp\htdocs\ekinerja\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('CodeIgniter\\Rou...')
#4 G:\xampp\htdocs\ekinerja\system\Config\Services.php(699): CodeIgniter\Autoloader\Autoloader->loadClass('CodeIgniter\\Rou...')
#5 G:\xampp\htdocs\ekinerja\system\Config\BaseService.php(123): CodeIgniter\Config\Services::routes(false)
#6 G:\xampp\htdocs\ekinerja\system\Config\Services.php(696): CodeIgniter\Config\BaseService::getSharedInstance('routes')
#7 G:\xampp\htdocs\ekinerja\app\Config\Routes.php(6): CodeIgniter\Config\Services::routes()
#8 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(780): require('G:\\xampp\\htdocs...')
#9 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(364): CodeIgniter\CodeIgniter->tryToRouteIt(NULL)
#10 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 G:\xampp\htdocs\ekinerja\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2021-04-12 07:09:28 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at G:\xampp\htdocs\ekinerja\app\Config\Events.php:33) in G:\xampp\htdocs\ekinerja\system\Debug\Exceptions.php:164
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'G:\\xampp\\htdocs...', 164)
#1 G:\xampp\htdocs\ekinerja\system\Debug\Exceptions.php(164): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-04-12 02:25:18 --> The framework needs the following extension(s) installed and loaded: {0}.
#0 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(224): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(172): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 G:\xampp\htdocs\ekinerja\system\bootstrap.php(159): CodeIgniter\CodeIgniter->initialize()
#3 G:\xampp\htdocs\ekinerja\index.php(36): require('G:\\xampp\\htdocs...')
#4 {main}
CRITICAL - 2021-04-12 07:28:50 --> Unable to connect to the database.
#0 G:\xampp\htdocs\ekinerja\system\Database\MySQLi\Connection.php(374): CodeIgniter\Database\BaseConnection->initialize()
#1 G:\xampp\htdocs\ekinerja\system\Database\BaseConnection.php(1413): CodeIgniter\Database\MySQLi\Connection->_escapeString('nci.ahmad@gmail...')
#2 G:\xampp\htdocs\ekinerja\system\BaseModel.php(1772): CodeIgniter\Database\BaseConnection->escapeString('nci.ahmad@gmail...')
#3 G:\xampp\htdocs\ekinerja\system\Model.php(832): CodeIgniter\BaseModel->__call('escapeString', Array)
#4 G:\xampp\htdocs\ekinerja\app\Models\Auth\LoginModel.php(31): CodeIgniter\Model->__call('escapeString', Array)
#5 G:\xampp\htdocs\ekinerja\app\Controllers\Pages\Auth\Auth.php(41): App\Models\Auth\LoginModel->cek('nci.ahmad@gmail...', 'trinifornia13')
#6 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(936): App\Controllers\Pages\Auth\Auth->signin()
#7 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pages\Auth\Auth))
#8 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 G:\xampp\htdocs\ekinerja\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-04-12 10:07:03 --> Undefined property: stdClass::$data
#0 G:\xampp\htdocs\ekinerja\app\Controllers\Pages\Auth\Auth.php(34): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'G:\\xampp\\htdocs...', 34, Array)
#1 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(936): App\Controllers\Pages\Auth\Auth->sso()
#2 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pages\Auth\Auth))
#3 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 G:\xampp\htdocs\ekinerja\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-04-12 10:11:06 --> Call to undefined method CodeIgniter\Session\Session::set_userdata()
#0 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(936): App\Controllers\Pages\Auth\Auth->sso()
#1 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pages\Auth\Auth))
#2 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 G:\xampp\htdocs\ekinerja\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-04-12 14:41:28 --> Undefined variable: link
#0 G:\xampp\htdocs\ekinerja\app\Controllers\Backend\Option\Users.php(163): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'G:\\xampp\\htdocs...', 163, Array)
#1 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(936): App\Controllers\Backend\Option\Users->memuatLink()
#2 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Backend\Option\Users))
#3 G:\xampp\htdocs\ekinerja\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 G:\xampp\htdocs\ekinerja\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
